### ISSUE TITLE
[Bug] When using bignum compilation fails

### ISSUE THREAD

**Issue by jtenner** [MaxGraey/bignum.wasm#18]
```
Error: Import file '~lib/internal/arraybuffer.ts' not found.
    at v (C:\Users\jtenner\Desktop\projects\near-runtime-ts\node_modules\assemblyscript\dist\asc.js:1:528413)
    at Object.exports.main (C:\Users\jtenner\Desktop\projects\near-runtime-ts\node_modules\assemblyscript\dist\asc.js:1:529177)
    at C:\Users\jtenner\Desktop\projects\near-runtime-ts\node_modules\as-pect\lib\cli\run.js:183:27
    at Array.forEach (<anonymous>)
    at Object.run (C:\Users\jtenner\Desktop\projects\near-runtime-ts\node_modules\as-pect\lib\cli\run.js:182:36)
    at Object.asp (C:\Users\jtenner\Desktop\projects\near-runtime-ts\node_modules\as-pect\lib\cli\index.js:68:19)
    at Object.<anonymous> (C:\Users\jtenner\Desktop\projects\near-runtime-ts\node_modules\as-pect\bin\asp:2:32)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
```

This is the result of https://github.com/MaxGraey/bignum.wasm/blob/a25aea823a65e843b0888e00707f2dce9ddd01b3/assembly/integer/u128.ts#L1

**Comment by MaxGraey** [comment]
Currently bignum not adopted to latest runtime. See this PR #17

**Comment by MaxGraey** [comment]
Fixed in #17
